packages/browser/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@rolldown/wasi",
```

Comment:
Please keep this package. We are using it in stackblitz: https://stackblitz.com/github/rolldown/rolldown-starter-stackblitz?file=package.json.